### DISCUSSION TITLE
nvfancontrol "Option Fanflicker"?

### DISCUSSION THREAD

**Discussion by istenburg** [foucault/nvfancontrol#43] [General]
Hello,
how and where do I set the option "fanflicker"?

**Comment by foucault** [comment]
Add the `--fanflicker` (or `-r`) option when starting the program. The format is LOWER,UPPER. For instance `-r 11,38` as already explained in the [README](https://github.com/foucault/nvfancontrol#fan-flicker-prevention).

**Comment by istenburg** [comment]
so I entered it like this

`[Desktop Entry]

Name=NV-Fancontrol
Comment[de]=GPU Lüfter steuern
Exec=/usr/bin/nvfancontrol -r 11,38
#Icon=/usr/share/pixmaps/tvbrowser128.png
Terminal=false
Type=Application
Categories=System;
StartupNotify=false`

but does not start nvfancontrol
